Classes/Domain/Model/Configuration.php

```diff
@@ -244,6 +245,7 @@ public function setEndTime($endTime)
      */
     public function getStartDate()
     {
+        $this->startDate->setTimezone(new \DateTimeZone(date_default_timezone_get()));
```

Comment:
Please use the Utility function:
https://github.com/lochmueller/calendarize/blob/master/Classes/Utility/DateTimeUtility.php#L75